app/helpers/application_helper.rb

```diff
@@ -385,6 +385,12 @@ def show_parent?(obj)
     (obj.new_record? && obj.class.count > 0) || (!obj.new_record? && obj.class.count > 1)
   end
 
+  def documentation_button(section)
+    link_to(icon_text('question-sign', _('Documentation'), :class => 'icon-white'),
+            "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html##{section}",
```

Comment:
I investigated symlinks at one point and it isn't possible with GitHub pages or jekyll (I forget which).  We could easily add a redirect into foreman-infra (e.g. https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/web/templates/web.conf.erb#L13) and add this into the [release process](http://projects.theforeman.org/projects/foreman/wiki/Release_Process) to change when the next manual is created.